src/view/com/composer/Composer.tsx

```diff
@@ -337,6 +337,7 @@ export const ComposePost = ({
 
   const onNewLink = useCallback(
     (uri: string) => {
+      dispatch({type: 'embed_add_uri', uri})
```

Comment:
Intentionally placed before the `return`. The logic for whether to handle this URI or ignore it is in the reducer now.

Reply:
May want a comment here? Feels like something someone might run over in the future

Reply:
Oh I get it, we're gonna remove the logic below 👍